refresh/index.js

```diff
@@ -78,6 +83,63 @@ module.exports = generators.Base.extend({
     }
   },
 
+  default: {
```

Comment:
Also, we've typically been placing actions in the part of the file where it matches the order of execution of the phases. So it would be better if these were moved to be with the other default phase actions like `parseFromSwagger`.